src/lbpDashboard/lbpDashboardForm/lbpDashboardForm.scss

```diff
@@ -58,16 +57,22 @@
           }
           .dropdown {
             width: 140px;
-
             .selectedLabel {
               height: 63px;
               border-top-right-radius: 0;
               border-bottom-right-radius: 0;
-              background-color: transparent;
+              background-color: $BG02;
             }
 
             .menu {
               border-top-right-radius: 0;
+              padding-top: 53px;
```

Comment:
I'm wondering, are any of these fixes general to "symbol-only" and thus can be included in tokenSelect.scss instead of here, so they can potentially apply to other times we might want to use symbol-only?

Reply:
I've moved some declarations around to make the symbolOnly element more generic:

Generic Use: ![image](https://user-images.githubusercontent.com/2517870/140622928-1fdbcb2a-1563-447b-b4bb-994b14db6257.png)  DashboardForm styling: ![image](https://user-images.githubusercontent.com/2517870/140622935-cfd2ecf4-5029-4f6a-a060-2ad540b253a4.png)